src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -245,6 +245,16 @@ export abstract class InstructorSessionBasePageComponent {
   }
 
   getCopyErrorMessage(): string {
+    const templateErrorString: string = Object.values(this.failedToCopySessions)[0];
+    if (templateErrorString.match(' exists already in the course ')) {
+      const courseNameStartIndex: number = templateErrorString.indexOf(' exists already in the course ')
+          + ' exists already in the course '.length;
+      return `${templateErrorString.substring(0, courseNameStartIndex)}
```

Comment:
There are too many magic entities being passed around here. What is the intention of this block?